src/type/__tests__/definition-test.js

```diff
@@ -1180,3 +1180,20 @@ describe('Type System: NonNull must only accept non-nullable types', () => {
     });
   }
 });
+
+describe('Type System: Configs', () => {
+  it('GraphQLObjectType have correct configs', () => {
+    const objectConfig = {
+      name: 'Object',
+      fields: {},
+      description: 'Graphql object type',
+      isTypeOf: () => true,
+      interfaces: [],
+      astNode: null,
+      extensionASTNodes: [],
+    };
+    const someObjectType = new GraphQLObjectType(objectConfig);
+
+    expect(someObjectType.toConfig()).to.deep.equal(objectConfig);
+  });
```

Comment:
Thanks for feedback. I could not figure out creating astNode. How can I create it?

Reply:
> How can I create it?

@doniyor2109 I need to think about it and also review your new utility functions.

Please keep in mind that tests shouldn't be more complex that functionality it testing.
It's totally normal in tests to break DRY and duplicate a couple of lines in every test if this will result in more self contain and easy to read test.

Reply:
@IvanGoncharov Thanks. I will try make it more readable.